src/Applications/GEOSdas_App/fvsetup

```diff
@@ -538,6 +538,20 @@ $ENV{"PATH"} = $FindBin::Bin .":$ENV{PATH}";
   $rc = system("$fvbin/fix_gocart_rc.csh $vres $fvhome/run/gocart");
   die "Failed to fix GOCART levels-referencing $!" if ( $rc );
 
+# LDAS Setup is done separately, follow instructions in GEOSldas/README.md   
+# ------------------------------------------------------------------------
+    print "\n checkprintout  ldas_ana == 1 ($ldas_ana)\n";
+    if ($ldas_ana == 1 ) {
+    print "\n Please run ldas_setup following instructions in GEOSldas/README.md\n";
+    print "\n requires config input files: YOUR_exeinp.txt, YOUR_batinp.txt.\n" ;
+    print "\n example LDROOT: /discover/nobackup/$user/GEOSldas\n" ;
+    print "\n example LDHOME: /discover/nobackup/$user/ldascoup_C180\n" ;
+    print "\n example cd $LDROOT/install/bin/ \n";
+    print "\n example salloc \n";
+    print "\n example source g5_modules \n" ;
+    print "\n example ldas_setup setup $LDHOME exp_exeinp.txt exp_batinp.txt \n" ;
```

Comment:
Do we really need these lines ?  This should be covered in `GEOSldas/README.md`.
Not sure about `salloc` command.
But most importantly, does the GEOSldas experiment need to be set up in a specific directory, or is there a place where we can tell the ADAS where the GEOSldas experiment is set up?  If the "example" directories have to be followed to the letter, we would need to say so here.

Reply:
LDROOT and LDHOME are needed at the time of fvsetup ( input to fvsetup).   The rest of the lines are not essential  and can be removed. 
 The path and names of "GESOldas"  and "ldascoup_C180" are just example, not to be followed.

Reply:
It doesn't seem to me that the sample directories for LDROOT and LDHOME are of much help here, so I would delete these lines as well.  Rather, the key message here should be that LDROOT and LDHOME need to be specified consistently in fvsetup and ldas_setup.  Can this message about ldas_setup be placed such that LDROOT and LDHOME are already known to fvsetup?  Then the message could print LDROOT and LDHOME and tell the user to specify these values in ldas_setup.  
If LDROOT and LDHOME end up in a config file somewhere, maybe we could also apply the "ldas_add2rc" consistency check to LDROOT and LDHOME?
Perhaps I'm missing something, but I have a sense that we can still make the interplay of fvsetup and ldas_setup more robust.

Reply:
at the time of fvsetup , ldas_setup is not yet run.   User has to have decision before run fvsetup on where his ldas root and  ldas exp home would be and give those info to fvsetup. ( and he should keep the same path/name  later when do ldas_setup ).   LDROOT and LDHOME setup by fvsetup  will end up in adas  top run script and put into env. variables during the run.

Reply:
Thanks for the clarification, and sorry I'm so slow to catch up.  From what I understand now, though, it looks like we could indeed **print this message about ldas_setup _after_ fvsetup has been given LDROOT and LDHOME**.  Then the message could echo the LDROOT and LDHOME values and tell the user that the exact same values need to be specified (under their GEOSldas names) in ldas_setup.  Does that make sense?